### PR TITLE
Upgrade to v1.27.0 of the OpenStack Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/dotnet/ContainerInfra/Cluster.cs [hunk 1]
```diff
@@ -153,7 +153,7 @@ public partial class Cluster : Pulumi.CustomResource
         public Output<string> Name { get; private set; } = null!;
 
         [Output("nodeAddresses")]
-        public Output<string> NodeAddresses { get; private set; } = null!;
+        public Output<ImmutableArray<string>> NodeAddresses { get; private set; } = null!;
```
Comment (mikhailshilkov) [inline sdk/dotnet/ContainerInfra/Cluster.cs:156]: More breaking changes similar to Linode...

Comment (stack72) [reply sdk/dotnet/ContainerInfra/Cluster.cs:156]: These are legit breaking changes in the TF provider - they are moving from a string to a list

https://github.com/terraform-providers/terraform-provider-openstack/commit/0db48f563e378b9feaeead4eec8c90ba7bf06c64

Comment (leezen) [reply sdk/dotnet/ContainerInfra/Cluster.cs:156]: Shouldn't this be a major version bump if we're going to introduce a breaking change?

Comment (stack72) [reply sdk/dotnet/ContainerInfra/Cluster.cs:156]: So the guidance we had used before (from Cyrus) was that it was only a major version bump if there was a risk of people's instances getting deleted. If there was a compile time change, we can minor bump - the same way that TF is doing

**Comment by stack72** [comment]
Closed via #86

**Pull request closed without merging**